Add datadog-crds chart (#78)
DataDog/helm-charts: charts/datadog-crds/README.md

# Datadog CRDs

![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1](https://img.shields.io/badge/AppVersion-1-informational?style=flat-square)

This chart was designed to allow others "datadog" charts to share `CustomResourcesDefinitions` such as the `DatadogMetric`.

## How to use Datadog Helm repository

You need to add this repository to your Helm repositories:

```
helm repo add datadog https://helm.datadoghq.com
helm repo update
```

## Prerequisites

> This chart can be use with Kubernetes `1.11+` or OpenShift `3.11+` in order to support `CustomResourcesDefinitions`.

Reviewer: ```suggestion
This chart can be used with Kubernetes `1.11+` or OpenShift `3.11+` due to the requirement to support `CustomResourcesDefinitions`.
```

> This chart can be used with Kubernetes `1.11+` or OpenShift `3.11+` to the requirement to support `CustomResourcesDefinitions`.

Reviewer: ```suggestion
This chart can be used with Kubernetes `1.11+` or OpenShift `3.11+` since  `CustomResourceDefinitions` are supported starting with these versions.
```

> But the recommanded Kubernetes version are `1.16+`.

Reviewer: ```suggestion
But the recommended Kubernetes versions are `1.16+`.
```
